doc/README_Sink.md

```diff
@@ -175,6 +177,33 @@ The following settings are used to configure the Cosmos DB Kafka Sink Connector.
 
 Data will always be written to the Cosmos DB as JSON without any schema.
 
+## Single Message Transforms
+
+Along with the Sink connector settings, you can specify the use of Single Message Transformations (SMTs) to modify messages flowing through the Kafka Connect platform. Refer to the [Confluent SMT Documentation](https://docs.confluent.io/platform/current/connect/transforms/overview.html) for more information.
+
+### Using SMTs to apply Time to live (TTL) for Cosmos DB items
```

Comment:
LOVE this! thank you for doing this. we should keep adding further SMT examples to this section as we find them.